{"name": "Проверка изменения данных неавторизованного пользователя", "status": "passed", "description": "Неавторизованный пользователь не может изменить своих данных", "start": 1723215315940, "stop": 1723215316159, "uuid": "864c3cc3-3726-427a-b35e-e7efa602d1b7", "historyId": "ef27a8e12292c8d78ca0fc8b4dd9339f", "testCaseId": "ef27a8e12292c8d78ca0fc8b4dd9339f", "fullName": "tests.test_user.TestChangingUserData#test_chang_user_data_not_auth_user_new_data_error_message", "labels": [{"name": "epic", "value": "test_changing_user_data"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user"}, {"name": "subSuite", "value": "TestChangingUserData"}, {"name": "host", "value": "DESKTOP-619AU3M"}, {"name": "thread", "value": "12804-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user"}]}